src/Symfony/Component/Console/Color.php

```diff
@@ -148,14 +148,35 @@ private function convertHexColorToAnsi(int $color): string
         $b = $color & 255;
 
         // see https://github.com/termstandard/colors/ for more information about true color support
```

Comment:
This comment should probably go in the phpdoc of the extracted method

Reply:
Well, it is. This comment was near the logic determining truecolor support before. You extracted the logic elsewhere without keeping the comment with it.